build/build-netlify.sh

```diff
@@ -51,10 +51,10 @@ if [[ "$INCOMING_HOOK_TITLE" = "nightly" ]]; then
 fi;
 
 # Run Algolia if building master
-# if [[ "$CONTEXT" = "production" ]]; then
-# 	echo "Building Algolia index..."
-# 	ALGOLIA_API_KEY=$PROD_ALGOLIA_API_KEY bundle exec jekyll algolia --config _config_base.yml,_config_url.yml --builds-config _config_cockroachdb.yml
-# fi;
+if [[ "$CONTEXT" = "production" ]]; then
```

Comment:
```suggestion
if [ "$CONTEXT" == "production" ]; then
```
The non-POSIX version of `test` always looks like a syntax error to me since `==` is usually used to test equality, and there's not much call for it unless you're doing regex pattern-match testing.